website/cue/reference/components/sources.cue

```diff
@@ -6,6 +6,14 @@ components: sources: [Name=string]: {
 	features: _
 
 	configuration: {
+		if features.collect != _|_ {
+				if features.collect.proxy != _|_ {
+					if features.collect.proxy.enabled {
+						proxy: base.components.sources.configuration.proxy
+					}
+				}
+		}
+
```

Comment:
@tobz, I think this was your suggestion - and it looked happy reviewing the site locally.